cmake/FindIceStorm.cmake

```diff
@@ -0,0 +1,19 @@
+set(icestorm_default_install_prefix ${CMAKE_INSTALL_PREFIX})
+if (DEFINED ICEBOX_ROOT) # for compatibility with old build scripts
```

Comment:
while we're refactoring things, we can probably just remove this, it's been deprecated for very many years now